cpp/src/io/parquet/page_delta_decode.cu

```diff
@@ -625,4 +777,30 @@ void __host__ DecodeDeltaByteArray(cudf::detail::hostdevice_vector<PageInfo>& pa
   }
 }
 
+/**
+ * @copydoc cudf::io::parquet::gpu::DecodeDeltaByteArray
+ */
+void __host__
```

Comment:
Why do we need to mark it `__host__`?
```suggestion
void __host__
```

Reply:
It's an idiosyncrasy I picked up from the original DecodePageData. I guess to be explicit about what's host code and what's device code when they're mixed?

Reply:
Yeah, a lot of older code has this. I don't include `__host__` in new code but I don't feel strongly about it. We should probably decide on the guideline and replace-all in one fell swoop.

Reply:
I count 7 instances of its use in the parquet code, 3 of which were added by me :sweat_smile: I think we can get rid of them all as part of this PR. Failing that I can at least get rid of the ones in page_delta_decode.cu